setup.cfg

```diff
@@ -56,7 +56,7 @@ install_requires =
     torch>1.9,<1.12
     requests
     pydantic
-    transformers==4.18.0
+    transformers==4.19.2
```

Comment:
I just checked the release notes and didn't see anything problematic. The only sentence that stood out was "Disclaimer: this release [v4.19.0] is the first release with no Python 3.6 support.". Haystack anyway supports only 3.7 or higher and we are not aware of anyone somehow still running it with python 3.6. So all good. 👍